src/api/ambassador_api.js

```diff
@@ -0,0 +1,74 @@
+import yoga from "../assets/yoga@2x.png"
+import { dashedToCamelObject } from "../utils/dashedToCamel";
+const wait = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+const ambassadors = [
+  {
+    id: 2,
```

Comment:
Both ambassadors in this list have `id: 2`. My first assumption would be that the route ID parameter (`/ambassador/1`) would filter the Ambassadors list and display the corresponding ID `ambassadors.filter({id: 1})`. In this scenario, `filter` would return 2 elements.